x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -487,5 +520,46 @@ export default function ({ getService }: FtrProviderContext) {
         }
       });
     });
+
+    describe('add trained model flyout for ML power user', () => {
+      before(async () => {
+        await ml.securityUI.loginAsMlPowerUser();
+        await ml.navigation.navigateToTrainedModels();
+        await ml.commonUI.waitForRefreshButtonEnabled();
+
+        await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+        await ml.trainedModelsFlyout.assertOpen(true);
+      });
+
+      after(async () => {
+        await ml.testExecution.logTestStep('Close the Add Trained Model flyout');
+        await ml.trainedModelsFlyout.close();
+
+        await ml.securityUI.logout();
+      });
+
+      it('should contain a Click to Download and a Manual Download tab', async () => {
+        await ml.testExecution.logTestStep(
+          'Assert the "Click to Download" and "Manual Download" tabs exists'
+        );
+        const clickToDownloadTab: AddModelFlyoutTabId = 'clickToDownload';
+        const manualDownloadTab: AddModelFlyoutTabId = 'manualDownload';
```

Comment:
these variables seem redundant. you can just pass strings as arguments, without specifying the type.